TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -331,10 +319,38 @@ import PromiseKit
         linkedBrokersToRemove.forEach { linkedBrokerToRemove in
             self.removeBroker(linkedBroker: linkedBrokerToRemove)
         }
-        
+
+        // Sync accounts
+        self.linkedBrokers.forEach { localLinkedBroker in
+            remoteLinkedBrokers.first { remoteLinkedBroker in
+                localLinkedBroker.linkedLogin.userId == remoteLinkedBroker.userId
+            }.flatMap { remoteLinkedBroker in
+                syncAccounts(localLinkedBroker: localLinkedBroker, remoteLinkedBroker: remoteLinkedBroker)
```

Comment:
We want also to update userToken (see comments in here : https://github.com/tradingticket/AndroidSDK/pull/63)